isa/Makefile

```diff
@@ -27,17 +27,32 @@ include $(src_dir)/rv32ud/Makefrag
 include $(src_dir)/rv32uzfh/Makefrag
 include $(src_dir)/rv32si/Makefrag
 include $(src_dir)/rv32mi/Makefrag
+include $(src_dir)/rv32uxpulpabs/Makefrag
+include $(src_dir)/rv32uxpulpbitop/Makefrag
+include $(src_dir)/rv32uxpulpbr/Makefrag
+include $(src_dir)/rv32uxpulpclip/Makefrag
+include $(src_dir)/rv32uxpulpmacsi/Makefrag
+include $(src_dir)/rv32uxpulpminmax/Makefrag
+include $(src_dir)/rv32uxpulppostmod/Makefrag
+include $(src_dir)/rv32uxpulpslet/Makefrag
+include $(src_dir)/rv32uxpulpvect/Makefrag
+include $(src_dir)/rv32uxpulpvectshufflepack/Makefrag
 
 default: all
 
 #--------------------------------------------------------------------
 # Build rules
 #--------------------------------------------------------------------
 
+MEMPOOL_DIR       := $(shell git rev-parse --show-toplevel 2>/dev/null || echo $$MEMPOOL_DIR)
+INSTALL_DIR       ?= $(MEMPOOL_DIR)/install
+GCC_INSTALL_DIR   ?= $(INSTALL_DIR)/riscv-gcc
+SPIKE_INSTALL_DIR ?= $(INSTALL_DIR)/riscv-isa-sim
+
 RISCV_PREFIX ?= riscv$(XLEN)-unknown-elf-
 RISCV_GCC ?= $(RISCV_PREFIX)gcc
 RISCV_GCC_OPTS ?= -static -mcmodel=medany -fvisibility=hidden -nostdlib -nostartfiles
-RISCV_OBJDUMP ?= $(RISCV_PREFIX)objdump --disassemble-all --disassemble-zeroes --section=.text --section=.text.startup --section=.text.init --section=.data
+RISCV_OBJDUMP ?= $(RISCV_PREFIX)objdump --disassembler-option="march=rv32gXpulpimg" --disassemble-all --disassemble-zeroes --section=.text --section=.text.startup --section=.text.init --section=.data
```

Comment:
Xpulpimg should not be here but we cannot remove it until we switch to a proper compiler (i.e. Robert's GCC with Xpulp sub-partitions). Can you add a TO-DO comment here for this?